nservicebus/upgrades/6to7/index.md

```diff
@@ -239,4 +239,9 @@ snippet: 6to7ConnectionStrings
 
 ## Installers
 
-NServiceBus Version 7 will only run installers when explicitly enabled via the `endpointConfiguration.EnableInstallers()` API. In previous versions, installers are automatically run when starting the endpoint with a debugger attached, this behavior has been removed in Version 7. Therefore, the `endpointConfiguration.DisableInstallers()` API has been obsoleted and is no longer required.
\ No newline at end of file
+NServiceBus Version 7 will only run installers when explicitly enabled via the `endpointConfiguration.EnableInstallers()` API. In previous versions, installers are automatically run when starting the endpoint with a debugger attached, this behavior has been removed in Version 7. Therefore, the `endpointConfiguration.DisableInstallers()` API has been obsoleted and is no longer required.
+
+
+## DistributionStrategy
+
+The `string SelectReceiver(string[] receiverAddresses)` signature has been removed from the `DistributionStrategy` base class. When writing a custom distribution strategy, implement the `string SelectDestination(DistributionContext context)` method instead which provides additional information usable for routing decisions. Receiver addresses can still be accessed via the `context.ReceiverAddresses` property.
```

Comment:
show a before+after snippet

Reply:
given the method is abstract and that I expect nobody outside particular to write a custom distribution strategy, I'm not sure this provides additional value.